src/Sylius/Bundle/TaxonomyBundle/Form/Type/TaxonomyType.php

```diff
@@ -32,6 +33,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'form_type' => 'sylius_taxonomy_translation',
                 'label'    => 'sylius.form.taxonomy.name'
             ))
+            ->addEventSubscriber(new BuildTaxonomyFormListener())
```

Comment:
You shoud define `BuildTaxonomyFormListener` as a service and inject it ino that class. You need to create a constructor and update the form definiftion.